README.md

```diff
@@ -26,10 +26,25 @@ db.put('foo', { bar: 'baz' }, (err) => {
 })
 ```
 
-## Config
+## Api
 
-Database config can be specified as a `postgres://` uri in the `location` string passed to the `PgDOWN` factory function. Otherwise, it will be pulled from the `PG*` environment variables (TODO: document this), falling back to the defaults in the `pg` client lib.
+#### `pgdown(location)`
 
+Creates a `PgDOWN` object with `location` which can take the following forms:
+
+* `dbname://username:password@host:port/tablename`
+* `/tablename`
```

Comment:
The `postgres://` part is actually the uri scheme, not the db name -- so it's always postgres. In the postgres:// scheme dbname is the first path component.

We should prob add some tests for this (and properly leverage the `parse-db-url` module -- I haven't bothered w/ this yet) before we document it though. The current impl uses the pg env vars (we should doc these too), and some `PGDOWN_TEST_*` env vars for setting the test db. It still feels kinda sloppy though -- could definitely use a fresh set of eyes.

FWIW the `location` param itself can only be used to set the database (first path component) and the table (the remaining path components, joined back together w/ `/`). I have some hacky prototype code that (partially) implements the `sublevel` API on top of this, but I've been trying to get the connection/pool shite tightened up before finalizing this (since we *really* want to make sure all sublevels share the same conn pool -- pg's pretty damn stingy w/ conn allocations since it forks on each one).